tests/director/external-services-mock-integration/auditlog_integration_test.go

```diff
@@ -0,0 +1,67 @@
+package external_services_mock_integration
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"net/http"
+	"testing"
+
+	"github.com/kyma-incubator/compass/tests/director/pkg/gql"
+	"github.com/kyma-incubator/compass/tests/director/pkg/idtokenprovider"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/kyma-incubator/compass/components/director/pkg/graphql"
+	"github.com/kyma-incubator/compass/tests/director/pkg/ptr"
+)
+
+func TestAuditlogIntegration(t *testing.T) {
+	ctx := context.Background()
+	httpClient := http.Client{}
+	appName := "app-for-testing-auditlog-mock"
+	appInput := graphql.ApplicationRegisterInput{
+		Name:         appName,
+		ProviderName: ptr.String("compass"),
+	}
+
+	t.Log("Get Dex id_token")
+	dexToken, err := idtokenprovider.GetDexToken()
```

Comment:
Can we use `Authenticate` method, which takes `config` as parameter. Because `GetDexToken` reads env inside.

Reply:
As discussed, it will be resolved with refactoring integration tests.